src/gateio_new_coins_announcements_bot/new_listings_scraper.py

```diff
@@ -47,10 +48,19 @@ def get_announcement():
     random.shuffle(queries)
     logger.debug(f"Queries: {queries}")
     request_url = (
-        f"https://www.binancezh.com/gateway-api/v1/public/cms/article/list/query"
+        f"http://www.binance.com/gateway-api/v1/public/cms/article/list/query"
         f"?{queries[0]}&{queries[1]}&{queries[2]}&{queries[3]}&{queries[4]}&{queries[5]}"
     )
-    latest_announcement = requests.get(request_url)
```

Comment:
I also made similar changes in #140, maybe you can integrate the changes here (mainly a better log message and same try-catch for kucoin announcement check)